### DISCUSSION TITLE
cvat annotation view always using compressed images

### DISCUSSION THREAD

**Discussion by kristofbruyninckx** [cvat-ai/cvat#3824] [Q&A] [answered]
When creating a task there are a few options to select image quality, chunk size and so on. 

Setting the image quality to 100% does not matter for the annotation view, which always seems to compress the image. I also set tried setting the chunk size to 1. and disabling the `force to use zip chunks as compressed data option`

In every case, cvat downloads compressed images for the annotation view, which generates artifacts that are difficult to discern from actual defects for my specific dataset. By using chrome dev tools i can see that a chunk of data is downloaded from the api like this `http://some_ip:8085/api/v1/tasks/10/data?type=chunk&number=0&quality=compressed`. Manually changing `compressed` to `original` here gives me a zip with data in original quality. Is there any way for the annotation view to pass this parameter and use original quality? After all i chose 100% quality for a reason.

**Comment by nmanovic** [comment, marked as answer]
@kristofbruyninckx , I agree that having the switch in UI is a good idea. If you can contribute the feature, it will be great. I believe it is enough to have a switch inside player settings.

**Comment by jbohnslav** [comment]
Is there any progress on this issue? I find that even if I upload a task with `--image_quality 100`, the annotation view is very compressed.

**Reply by O-J1** [reply]
This is still an issue

**Comment by merlinwizzard** [comment]
fix all repositories

**Comment by jaroslavknotek** [comment]
Is there any workaround? I have images which are impossible to even see:

Input image(cropped):
![image](https://github.com/user-attachments/assets/53c6628e-7da3-409a-9351-fd76ee651f5d)


Image in the editor:
![image](https://github.com/user-attachments/assets/b9258a99-f24a-4e47-ab0a-1cb26ff7605e)


I am not able to contribute in any way, however, I tried to "hard code" `original` compression in the code but it sees to be on ~30 different places. Is there any patch for this for the `develop` branch?
